doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md

```diff
@@ -15,6 +15,7 @@
   Listing, grid, tree view are not severely affected as the main permission is checked on a Kernel event level that prevents the page to be shown and prevents any process that iterate isAllowed() calls. 
   The only cases could be affected are those where the workspace are set but master permissions are disallowed, before this change, it could lead to (not intended) false positive.
 - [Security/User] `UsernameNotFoundException` (deprecated since Symfony 5.3) occurences have been replaced with `UserNotFoundException`.
+- [Deprecated] Generate type declarations option in class definition is deprecated, because type declarations will be added with Pimcore 11
```

Comment:
```suggestion
- [Deprecated] Generate type declarations option in class definition is deprecated, because type declarations will always be added with Pimcore 11
```